js/publish/src/ui/styles/player.css

```
@@ -0,0 +1,114 @@
+	line-height: 1.4;
+}
+
+.player--chrome .chrome {
+	opacity: 1;
+	transform: translateY(0);
+	pointer-events: auto;
+}
+
+/* Fullscreen. */
+.player:fullscreen,
+.player:-webkit-full-screen,
```

Stylelint (17.12.0)

[error] Vendor-prefixed selector ":-webkit-full-screen" (selector-no-vendor-prefix)

(selector-no-vendor-prefix)